src/conf.py

```diff
@@ -116,6 +116,8 @@ def json_load(path):
 }
 CDN = 'https://' + CDNS_BY_ENV.get(ENV, DEFAULT_CDN)
 
+IIIF = 'https://prod--iiif.elifesciences.org/lax:%(padded-msid)s/%(fname)s'
```

Comment:
Great point - I was mostly just checking for validation on this first run, and this detail will be added shortly.